src/Card/Card.scss

```diff
@@ -173,6 +173,7 @@
 }
 
 .pgn__card-footer-text {
+  width: 100%;
```

Comment:
I'm not sure we can set `width: 100%` here in all situations, as it messes with the styling [elsewhere](https://deploy-preview-1472--paragon-openedx.netlify.app/components/card/#vertical-variant).

<img width="981" alt="image" src="https://user-images.githubusercontent.com/2828721/181762487-c3756a8d-f653-42ce-a752-596a23166c26.png">

Reply:
Corrected